lib/taskana-core-test/src/test/java/acceptance/TaskanaConfigurationTest.java

```diff
@@ -289,6 +289,8 @@ void should_PopulateEveryTaskanaConfiguration_When_EveryBuilderFunctionIsCalled(
       boolean expectedAddAdditionalUserInfo = true;
       Set<WorkbasketPermission> expectedMinimalPermissionsToAssignDomains =
           Set.of(WorkbasketPermission.CUSTOM_2);
+      // database configuration
+      boolean expecteduseSpecificDb2Taskquery = false;
```

Comment:
Should we maybe make the "u" in "use" a capital letter to follow camelCase?

Reply:
Sounds good :)